File: app/TitikBanjir.php
<?php

namespace App;

use GoldSpecDigital\LaravelEloquentUUID\Database\Eloquent\Model;

class TitikBanjir extends Model
{

    protected $fillable = [
        'id_admin', 'geometry', 'nama_jalan', 'kondisi_kerusakan', 'foto','keterangan'
    ];
    protected $spatialFields = [
        'geometry'
    ];
    protected $hidden = [
        'id_admin',
    ];
}

File: app/src/main/java/com/azizapp/test/model/TitikBanjir.kt
package com.azizapp.test.model


import android.annotation.SuppressLint
import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@SuppressLint("ParcelCreator")
@Parcelize
data class TitikBanjir(
    @SerializedName("id")
    val id: String,
    @SerializedName("nama_jalan")
    val namaJalan: String,
    @SerializedName("kondisi_kerusakan")
    val kondisiKerusakan: String,
    @SerializedName("foto")
    val foto: String,
    @SerializedName("keterangan")
    val keterangan: String,
    @SerializedName("geometry")
    val geometry: String
) : Parcelable